tools/pkg/repo/create.py

```diff
@@ -380,6 +380,9 @@ def rpm(
         assert incoming is not None
         assert repo_path is not None
         assert key_id is not None
+
+    if distro == "photon":
+        distro_version = f"{distro_version}.0"
```

Comment:
can we always rely on '.0' , probably not

Reply:
The photon team has indicated that they have never used that versioning. they put a .X in place if they ever wanted to, but functionally never have.